### ISSUE TITLE
no-install mode panicked

### ISSUE THREAD

**Issue by qt911025** [rustwasm/wasm-pack#553]
## 🐛 Bug description
Panicked while running `wasm-pack build --mode no-install`.

Due to my network condition, wasm-pack can't download wasm-bindgen from github, which is on the blocked AWS server and won't redirect to another available source like downloading by browser.(Btw, the installation script `rustwasm.github.io/wasm-pack/installer/init.sh` has this problem too.)

So I installed `wasm-bindgen-cli` manually. `wasm-pack build` functioned but `wasm-pack build --mode no-install` panicked. I can just use the former but the latter is weird, isn't it?

#### 🤔 Expected Behavior
`wasm-pack build --mode no-install` should function well.

#### 👟 Steps to reproduce
1. Install `wasm-pack` and `wasm-bindgen-cli` through cargo.
2. Initialize a project.
3. run `wasm-pack build --mode no-install`.

#### Crash report
```toml
name = 'wasm-pack'
operating_system = 'unix:Ubuntu'
crate_version = '0.6.0'
explanation = '''
Panic occurred in file 'src/libcore/option.rs' at line 355
'''
method = 'Panic'
backtrace = '''
stack backtrace:
   0: backtrace::backtrace::trace_unsynchronized::h7d3ac4163d4e3888 (0x55777f862ec6)
   1: backtrace::capture::Backtrace::create::hffdb9eb1f28c65a1 (0x55777f861e97)
   2: backtrace::capture::Backtrace::new::h98240d81a82dafc0 (0x55777f861dbd)
   3: human_panic::report::Report::new::ha2fbf09260075802 (0x55777f7be472)
   4: human_panic::handle_dump::h42af556617c8b15e (0x55777f7bd957)
   5: wasm_pack::main::{{closure}}::h56fd20a08b1fb415 (0x55777f58d458)
   6: std::panicking::rust_panic_with_hook::h8cbdfe43764887be (0x55777f88fdc9)
             at src/libstd/panicking.rs:495
   7: std::panicking::continue_panic_fmt::h3d3c5a833c00a5e1 (0x55777f88f871)
             at src/libstd/panicking.rs:398
   8: rust_begin_unwind (0x55777f88f755)
             at src/libstd/panicking.rs:325
   9: core::panicking::panic_fmt::h4d67173bc68f6d5a (0x55777f8ab63c)
             at src/libcore/panicking.rs:95
  10: core::panicking::panic::h6f50c0de2dcd7974 (0x55777f8ab56b)
             at src/libcore/panicking.rs:59
  11: wasm_pack::command::build::Build::step_run_wasm_bindgen::hed2b3d472c7902c6 (0x55777f6155ef)
  12: wasm_pack::command::build::Build::run::h9d3fef03647a78fb (0x55777f614075)
  13: wasm_pack::command::run_wasm_pack::h14ffd835658fe72f (0x55777f5938af)
  14: wasm_pack::main::he4a66eca2247468a (0x55777f58cfce)
  15: std::rt::lang_start::{{closure}}::h0424aeadad3beac6 (0x55777f58de92)
  16: std::rt::lang_start_internal::{{closure}}::hdfc28107b5be47c9 (0x55777f88f6f2)
             at src/libstd/rt.rs:59
      std::panicking::try::do_call::h69790245ac2d03fe
             at src/libstd/panicking.rs:310
  17: __rust_maybe_catch_panic (0x55777f8a51b9)
             at src/libpanic_unwind/lib.rs:102
  18: std::panicking::try::h9c1cbc5599e1efbf (0x55777f890183)
             at src/libstd/panicking.rs:289
      std::panic::catch_unwind::h0562757d03ff60b3
             at src/libstd/panic.rs:398
      std::rt::lang_start_internal::h540c897fe52ba9c5
             at src/libstd/rt.rs:58
  19: main (0x55777f58d4d1)
  20: __libc_start_main (0x7f9a906cc09a)
  21: _start (0x55777f587019)
  22: <unknown> (0x0)'''

```

#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version: 0.6.0
rustc version:1.32.0

**Comment by ashleygwilliams** [comment]
@qt911025 a fix for this was released in 0.7.0 - can you confirm that it fixed your problem?

**Comment by qt911025** [comment]
Yes, it fixed!:smile:
